pages/githubissues.md

```diff
@@ -7,8 +7,16 @@
 * Work with admins to make adjustments if necessary, and merge your branch into the upstream
 
 ## Introduction
+
+On GitHub, each repository has a section where issues can be added, discussed, and fixed as a means of categorizing and addressing problems we find. Issues are often opened by our team to fix software bugs on the BeLL and in our code.
+
+**Note**: Issues are not just about adding to the wiki, but can also involve removing repetitive or unnecessary code or content. 
+
+Below, you'll find an example of an issue where I brought up a problem, researched an answer, and then fixed the issue.
```

Comment:
I think I just added a space between those lines for appearance sake. That sentence was already in there.